src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -138,6 +142,6 @@ public void clear() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof UniquePersonList // instanceof handles nulls
-                        && this.internalList.equals(((UniquePersonList) other).internalList));
+                && this.internalList.equals(((UniquePersonList) other).internalList));
```

Comment:
Note that the indentation for wrapped lines is 8 spaces, following the Java coding standard.